dsc/tests/dsc_include.tests.ps1

```diff
@@ -183,4 +174,38 @@ resources:
         $out.results[1].result[0].result[0].type | Should -Be 'Test/Echo'
         $out.results[1].result[0].result[0].result[0].actualState.output | Should -Be 'one'
     }
+
+    It 'Set with include works' {
+        $echoConfig = @'
+$schema: https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2024/04/config/document.json
+resources:
+- name: one
+  type: Test/Echo
+  properties:
+    output: Hello World
+'@
+
+        $echoConfigPath = Join-Path $TestDrive 'echo.dsc.yaml'
+        $echoConfig | Set-Content -Path $echoConfigPath -Encoding utf8
+        # need to escape backslashes for YAML
+        $echoConfigPathParent = (Split-Path $echoConfigPath -Parent).Replace('\', '\\')
+        $echoConfigPathLeaf = (Split-Path $echoConfigPath -Leaf).Replace('\', '\\')
+        $directorySeparator = [System.IO.Path]::DirectorySeparatorChar.ToString().Replace('\', '\\')
+
+        $includeConfig = @"
+`$schema: https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2024/04/config/document.json
+resources:
+- name: nested
+  type: Microsoft.DSC/Include
+  properties:
+    configurationFile: "[concat('$echoConfigPathParent', '$directorySeparator', '$echoConfigPathLeaf')]"
```

Comment:
why using `concat` instead of using just full path, aka `$echoConfigPath` ?

Reply:
It was to test that expressions get resolved for Include